### PR TITLE
test: document code coverage generation

### PR DESCRIPTION
**Pull request by jelly**

### CODE REVIEW

File: test/README.md [hunk 1]
```diff
@@ -248,6 +248,31 @@ each section returns the machine to more or less the state that it was
 in before the section.  This makes it easier to run these sections
 ad-hoc when doing incremental development.
 
+## Coverage
+
+Every pull request will trigger a `$DEFAULT_OS/devel` scenario which creates a
+coverage report of the JavaScript code executed and writes comments about
+uncovered code in the pull request. The overall coverage percentage is recorded
+in prometheus for a subset of our projects and [visualized in
+Grafana](https://grafana-cockpit.apps.ocp.cloud.ci.centos.org/d/ci/cockpit-ci?orgId=1).
```
Comment (martinpitt) [inline test/README.md:257]: This seems to render okay, but looks weird, and may trip up some editors. Can you avoid the line break in the link?

File: test/README.md [hunk 2]
```diff
@@ -248,6 +248,31 @@ each section returns the machine to more or less the state that it was
...
+
+```
+export NODE_ENV=devel
+./build.js
+./test/image-prepare -q
+./test/common/run-tests --test-dir test/verify --coverage TestApps
+```
+
+Code which is impossible or very hard to test in our tests can be excluded from
+appearing in a pull request as comment by add a `not-covered` comment:
```
Comment (martinpitt) [inline test/README.md:269]: add*ing* and "... with a short justification".

**Review by martinpitt — APPROVED** [review]
Dankjewel!

**Comment by jelly** [comment]
testing-farm:centos-stream-9-aarch64 — Tests failed ... 

Unrelated to documentation.

**Pull request merged**